packages/@okta/vuepress-site/docs/guides/terraform-create-custom-auth-server/main/index.md

```diff
@@ -0,0 +1,407 @@
+
+```hcl
+resource "okta_auth_server_scope" "create_obj" {
+  auth_server_id = okta_auth_server.api_server.id
+  metadata_publish = "NO_CLIENTS"
+  name             = "apicreate"
+  consent          = "REQUIRED"
+}
+
+resource "okta_auth_server_scope" "read_obj" {
+  auth_server_id = okta_auth_server.api_server.id
+  metadata_publish = "NO_CLIENTS"
+  name             = "apiread"
+  consent          = "REQUIRED"
+}
+
+resource "okta_auth_server_scope" "replace_obj" {
+  auth_server_id = okta_auth_server.api_server.id
+  metadata_publish = "NO_CLIENTS"
+  name             = "apireplace"
+  consent          = "REQUIRED"
+}
+```
+
+### Add an authorization server policy
+
+Create at least one policy that specifies how to mint the tokens with the correct scopes based on your defined rules.
+
+Create an authorization server policy:
+
+1. Create an `okta_auth_server_policy` resource.
+
+1. Set `name` and `description` to the policy name and policy's description.
+
+1. Set `auth_server_id` to your authorization server ID.
+
+1. Set `status` to `ACTIVE`.
+
+1. Set `client_whitelist` to an array of the client IDs that use this policy. Pass the special array value `["ALL_CLIENTS"]` to use with all apps. To enable different policies for different apps, create multiple authorization server policies.
+
+1. If this is your only authorization server policy, set `priority` to `1`.
+
+   For more than one authorization server policy, set the `priority` attribute to `1`, `2`, `3`, and so on. To ensure reliable deployment of multiple policies, you must force Terraform resource creation order to match the `priority` attribute. Add the `depends_on` Terraform meta attribute to the policy with priority `2` so it depends on the one with priority `1`, and so on. For more information, see [Managing policy and rule dependencies](/docs/guides/terraform-syntax-tips/main/#managing-policy-and-rule-dependencies).
+
+For example:
+
+```hcl
+resource "okta_auth_server_policy" "example" {
+  auth_server_id = okta_auth_server.api_server.id
+  status           = "ACTIVE"
+  name             = "My OIDC App Authorization Policy"
+  description      = "example OIDC App Authorization Policy"
+  priority         = 1
+  client_whitelist = [okta_app_oauth.MyOIDCApp.id]
+}
+```
+
+### Add authorization server policy rules
+
+1. Create an `okta_auth_server_policy_rule` resource.
+
+1. Set `name` to a human-readable rule name.
+
+1. Set `auth_server_id` to the authorization server ID.
+
+1. Set `policy_id` to the policy ID.
+
+1. Set `status` to `ACTIVE`.
+
+1. Set `scope_whitelist` to the list of allowed scopes for this rule's app. Specify them as an array of IDs in the scope resource.
+
+   Optionally, you can pass the special value `["*"]` to grant all custom scopes defined in Terraform resources. Okta recommends using caution, especially if there may be multiple app integrations with different scopes in the future.
```

Comment:
This is the only time you refer to "app integrations" in the article. Easiest to just delete integration here for consistency.

```suggestion
   Optionally, you can pass the special value `["*"]` to grant all custom scopes defined in Terraform resources. Okta recommends using caution, especially if there may be multiple apps with different scopes in the future.
```

Reply:
in next commit